### PR TITLE
First draft of OSG 23 initial release

### PR DESCRIPTION
**Pull request by brianhlin**
A few items:

1.  ✅ This doesn't include `mkdocs.yml` changes so that we can publish ahead of the release
1.  ✅ We should update the HTCondor-CE section with a link to the HTCondor-CE 23 release section when that's ready
1.  We should add feature series + HTCondor-CE doc links in the bullets at the top as they're ready
1.  ✅ We have `oidc-agent` in development right now but we should just pull it.
    If this is the wrong decision, we can always release it next week.

### CODE REVIEW

**Comment by brianhlin** [comment]
We'll also want to update the `release_series.md` doc to include OSG 23 here https://osg-htc.org/docs/release/release_series/#series-overviews

File: docs/release/osg-23.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    -   This release completes EL9 and Python 3.9 support
+    -   Added structured logging
+    -   Various OSG_Autoconf improvements
+    -   Fixed bugs with Python 3.9 and rrdtools failures with missing ClassAds and monitoring
+
+#### Package Removals ####
+
+The following packages were removed from OSG 23 compared to OSG 3.6:
+
+-  `blahp`: available as `condor-blahp`
```
Comment (timtheisen) [inline docs/release/osg-23.md:136]: No more condor-blahp package. blahp components are in the main condor package.

Comment (brianhlin) [reply docs/release/osg-23.md]: Fixed

**Review by timtheisen — APPROVED** [review]
LGTM

**Pull request merged**